.travis.yml

```
@@ -11,6 +11,7 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9
```

```suggestion
  - 3.9-dev
```
`3.9` does not exist yet on travis-ci